### PR TITLE
Update error message for email formatting in cerberus schema check

### PR DESCRIPTION
**Pull request by eroberts9789**
Instead of writing separate custom validator, check for email formatting error in virtool.http.schema and update error message

### CODE REVIEW

**Comment by igboyes** [comment]
I think I may have misdirected you on this one. Please see the `check_with` rule in Cerberus and the existing module: https://github.com/virtool/virtool/blob/develop/virtool/validators.py.

This is the preferred solution for this issue.

File: virtool/validators.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import virtool.users.utils
```
Comment (igboyes) [inline virtool/validators.py:3]: Can you please update this module with the following:
- tests for all validators
- docstrings and typing
- `from` imports

File: virtool/validators.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
...
 
 RE_HEX_COLOR = re.compile("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
+RE_EMAIL = re.compile("^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
```
Comment (igboyes) [inline virtool/validators.py:6]: Did you review the `email-validator` package?

**Review by igboyes — CHANGES_REQUESTED** [review]

File: virtool/validators.py (hunk 2 above)
Comment (eroberts9789) [reply virtool/validators.py:6]: I did, I wasn't sure if you would prefer me to use that or not though

**Review by igboyes — APPROVED** [review]

**Pull request merged**